sycl/include/sycl/ext/oneapi/experimental/group_sort.hpp

```diff
@@ -92,15 +92,17 @@ sort_over_group(experimental::group_with_scratchpad<Group, Extent> exec,
                 T value, Compare comp) {
   return sort_over_group(
       exec.get_group(), value,
-      experimental::default_sorter<Compare>(exec.get_memory(), comp));
+      // experimental::default_sorter<Compare>(exec.get_memory(), comp));
```

Comment:
Seems, this is an extra line. Do you plan to remove it?

Reply:
Removed